### ISSUE TITLE
Game Freezes

### ISSUE THREAD

**Issue by ghost** [ruffle-rs/ruffle#12280]
### Describe the bug

If I open "Main.swf" it freezes and this error is in the terminal: ERROR ruffle_core::display_object::movie_clip: Got "ReferenceError: Error #1069: Property flash.desktop::NativeApplication not found on global and there is no default value." when constructing AVM2 side of movie clip of type Main.
I think this is because there are more than one swf file or it could be because its originally an Android app or because it requieres Adobe AIR to work.

### Expected behavior

Starts normally

### Affected platform

Desktop app

### Operating system

Windows 11

**Comment by Lord-McSweeney** [comment]
You are correct that Ruffle doesn't support AIR yet, and that is the cause of the issue.

Please provide a link to the SWF, or ZIP it and upload it to github. While Ruffle has no AIR support yet, it might be added in the future. In the meantime it would be useful to have some AIR SWF files for testing.

**Comment by Lord-McSweeney** [comment]
Closing as a SWF was not provided
